.travis.yml

```diff
@@ -4,11 +4,16 @@ scala:
 - 2.10.6
 
 before_install:
- - export PATH=${PATH}:./vendor/bundle
+- export PATH=${PATH}:./vendor/bundle
 
 install:
-  - gem install jekyll -v 2.5
+- gem install jekyll -v 2.5
```

Comment:
Jekyll latest version is [3.2.1](https://rubygems.org/gems/jekyll). IS there a reason why we are using an outdated version?